specification/compute/resource-manager/Microsoft.Compute/stable/2021-08-01/disk.json

```diff
@@ -2498,6 +2520,10 @@
           {
             "value": "EncryptionAtRestWithPlatformAndCustomerKeys",
             "description": "Resource using diskEncryptionSet would be encrypted at rest with two layers of encryption. One of the keys is Customer managed and the other key is Platform managed."
+          },
+          {
+            "value": "ConfidentialVmEncryptedWithCustomerKey",
+            "description": "Confidential VM supported disk encrypted with customer managed key."
```

Comment:
please be more descriptive similar to above. Would this encrypt only the guest state or can be used for disks too?

Reply:
checked with PM, and modified to -
"Confidential VM supported disk and VM guest state would be encrypted with customer managed key."